modules/gitops-argo-cd-installation.adoc

```diff
@@ -23,4 +23,16 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 .. Create an external OS Route to access Argo CD server. Click *Server* -> *Route* and check *Enabled*.  
 
-. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
\ No newline at end of file
+. To open the Argo CD web UI, click the route by navigating to *Networking* -> *Routes* -> *<instance name>-server* in the project where the Argo CD instance is installed.
++
+[NOTE]
+====
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally. 
+====
+. To log in with your username and password, obtain the password for the Argo CD instance:
+.. In the left panel of the console, use the perspective switcher to switch to the *Developer* perspective.
```

Comment:
This should be viewed in **Administrator** perspective

Reply:
Fixed.